datastore/database.go

```diff
@@ -67,15 +67,31 @@ type Datastore interface {
 	GetAccount(authorityID string) (Account, error)
 	UpdateAccountAssertion(authorityID, assertion string) error
 	PutAccount(account Account) (string, error)
+
+	CreateOpenidNonceTable() error
+	CreateOpenidNonce(nonce OpenidNonce) error
 }
 
 // DB local database interface with our custom methods.
 type DB struct {
 	*sql.DB
 }
 
+// Env Environment struct that holds the config and data store details.
+type Env struct {
+	Config    config.Settings
+	DB        Datastore
+	KeypairDB *KeypairDatabase
+}
+
+// Environ contains the parsed config file settings.
+var Environ *Env
+
+// OpenidNonceStore contains the database nonce store for Openid
+var OpenidNonceStore PgNonceStore
```

Comment:
The way the var is used causes a nil pointer exception at start-up if it is part of Environ.